test/System/Clock.cpp

```diff
@@ -0,0 +1,32 @@
+#include <SFML/System/Clock.hpp>
+
+#include <doctest.h>
+#include <chrono>
+#include <ostream>
+#include <thread>
+
+namespace sf
+{
+std::ostream& operator <<(std::ostream& os, const sf::Time& time);
```

Comment:
Is this defined anywhere?

Reply:
Yes in test/System/Time.cpp

Reply:
This is where a utilities header would make more sense...

Reply:
Is there a reason this operator isn't part of the library itself?

Reply:
> Is there a reason this operator isn't part of the library itself?

You can't really assume that everybody wants to print `sf::Time` in the same way, and also you would introduce a dependency on `<ostream>`. I wouldn't add this operator to the library.

Reply:
Fair point. Fixed.